docs/Users/Tutorials/Linter_Bears.rst

```diff
@@ -205,6 +205,10 @@ We can test our bear like this
     by their **filename** and display them based on their
     **classname**.
 
+Normally, providing a severity-map is not needed, as *coala* has a default
+severity-map which recognizes many common words used for severities. Check out
+the API documentation for keywords supported!
```

Comment:
how do I link to the api docs? especially to a specific class?
btw the linter API docs are still not working, so for now it makes no difference, I think we should file an issue for that^^